incentives-model/README.md

```diff
@@ -0,0 +1,14 @@
+# incentives-model
+
+Model to estimate token depletion from the Common Pool, using various parameters.
+
+## Running
+We recommend executing the app within a virtual environment, like conda. To
+initiate the app, use the commands provided below:
+
+```
+conda create --name inc python=3
+conda activate inc
+pip install -r requirements.txt
+python app.py
```

Comment:
```suggestion
$ python app.py
$ # or
$ flask run
```
Note that the `flask run` command will only work after the file `app.py` is amended, as suggested in the corresponding review comment.